python/lsst/ts/m2com/utility.py

```diff
@@ -20,10 +20,32 @@
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
 import json
+import yaml
+from copy import deepcopy
 
 from lsst.ts import tcpip
 
-__all__ = ["write_json_packet", "check_queue_size"]
+__all__ = [
+    "write_json_packet",
+    "check_queue_size",
+    "read_yaml_file",
+    "collect_queue_messages",
+    "get_queue_message_latest",
+    "TEST_DIGITAL_OUTPUT_NO_POWER",
+    "TEST_DIGITAL_OUTPUT_POWER_COMM",
+    "TEST_DIGITAL_OUTPUT_POWER_COMM_MOTOR",
+    "TEST_DIGITAL_INPUT_NO_POWER",
+    "TEST_DIGITAL_INPUT_POWER_COMM",
+    "TEST_DIGITAL_INPUT_POWER_COMM_MOTOR",
+]
+
+TEST_DIGITAL_OUTPUT_NO_POWER = 0x1C
```

Comment:
You sure you want to have these `TEST_DIGITAL_*` constants here in the `utility` sub-module? You could create a `constants` sub-module

Reply:
Fixed in the commit: [8e6d00e](https://github.com/lsst-ts/ts_m2com/pull/2/commits/8e6d00e84f176e5c1227e83f56e9a124aa1afd53). Thanks!